gog/graph.go

```diff
@@ -158,6 +158,15 @@ func (v *astVisitor) Visit(node ast.Node) (w ast.Visitor) {
 			return nil
 		}
 
+	case *ast.SelectorExpr:
+		if sel := v.typesInfo.Selections[n]; sel != nil {
+			t := sel.Recv()
+			if ptr, ok := t.(*types.Pointer); ok {
+				t = ptr.Elem()
```

Comment:
is it possible to have pointers to pointers? ie do we need to keep unwrapping pointers?